wpengine/atlas-shopify-blueprint
# [TITAN-341] - Create ACM Manifest for Shopify Blueprint

## Description

This change adds the blueprint manifest exported from ACM which will instruct Atlas how to build the WordPress configuration and content. 

## Testing

1. Create a fresh site in [Local](https://localwp.com/releases/) or your favourite dev environment.
2. Choose "go to site folder" and navigate to the "public" directory
3. Copy the `acm-blueprint.zip` file from this repo into the "public" directory
4. Back in Local, choose “Open Site Shell” to open a terminal and run WP CLI commands.
5. Install required plugins:
    ```
    wp plugin install --activate atlas-content-modeler faustwp wp-graphql
    ```
6. Clear default posts and pages:
    ```
    wp acm reset --yes --all
    ```
7. Import the current shopify blueprint zip that you copied into the "public" directory previously
    ```
    wp acm blueprint import acm-blueprint.zip
    ```
8. in WP Admin, confirm that the WordPress install has the correct settings and content as the demo store -  go a step further and connect this repos env vars to this new WordPress install and then do `npm run dev` where the site should work as expected
